src/xdocs/config_misc.xml

```diff
@@ -28,8 +28,16 @@
         <p>
           Checks the style of array type definitions.  Some like Java style:
           <code>public static void main(String[] args)</code> and some like
-          C style: public static void main(String args[])
+          C style: <code>public static void main(String args[])</code>.
         </p>
+        <p>
+          This check also enforces the style of method return types:
```

Comment:
`the style of` => `only Java style for`.
`:` => `.`
Make similar update to javadoc.